sharding-core/sharding-core-parse/sharding-core-parse-test/src/test/resources/sharding/dml/insert.xml

```diff
@@ -665,4 +665,79 @@
             </and-condition>
         </sharding-conditions>
     </parser-result>
+
+    <parser-result sql-case-id="insert_with_duplicate_key_way_1" parameters="1, 2, 3">
```

Comment:
Do not name as `xxx_1` and `xxx_2`, please name the test case make sense.

Reply:
I 'll  modify them.